util.cpp

```diff
@@ -79,6 +79,29 @@ namespace Sass {
     return out;
   }
 
+  // read css string (handle multiline DELIM)
+  string read_css_string(const string& str)
+  {
+    string out("");
+    bool esc = false;
+    for (auto i : str) {
+      if (i == '\\') {
+        esc = ! esc;
+      } else if (esc && i == '\r') {
```

Comment:
Actually it should only remove escaped newlines (a backslash before an actual linefeed). The implementation should be pretty correct and should support windows linefeeds to (but not old mac style). There is IMO only a very small "issue", that it will also actually remove `/\\\r*\n/`.

From https://github.com/sass/sass/issues/1237#issuecomment-42056762

> FYI http://www.w3.org/TR/CSS21/syndata.html#characters
>
> First, inside a string, a backslash followed by a newline is ignored (i.e., the string is deemed not to contain either the backslash or the newline). Outside a string, a backslash followed by a newline stands for itself (i.e., a DELIM followed by a newline).
>